lib/util/cli.js

```diff
@@ -85,7 +86,7 @@ const Cli = {
      */
     async getCredentialObject(properties, target, isCredentialOnly, allowAll) {
         try {
-            if (!(await Util.checkProperties(properties))) {
+            if (!(await Config.checkProperties(properties))) {
```

Comment:
does this logic still work, given that you run checkProperties inside of getProperties already?

Reply:
I assume you mean, is it needed? I would say actually no, so we can throw errors earlier on getProperties?